feat: be able to use build config

cdrage opened the pull request:
feat: be able to use build config

### What does this PR do?

* Under advanced have build config file option
* Be able to pass in YAML or JSON file
* Misc: Fixes size of font for AWS options

### Screenshot / video of UI

<!-- If this PR is changing UI, please include
screenshots or screencasts showing the difference -->

![Screenshot 2024-07-09 at 2 00 25 PM](https://github.com/containers/podman-desktop-extension-bootc/assets/6422176/3c41c29e-78ad-4032-8e2b-d4485fac73db)



### What issues does this PR fix or reference?

<!-- Include any related issues from Podman Desktop
repository (or from another issue tracker). -->

Closes https://github.com/containers/podman-desktop-extension-bootc/issues/471

### How to test this PR?

1. Use our test bootc image
2. Create a TOML file with the following:

```
[[customizations.user]]
name = "root"
password = "letmein"
```

3. Build image
4. Confirm you can login
5. Test again with a JSON file:
```
{
  "customizations": {
    "user": [
      {
        "name": "root",
        "password": "letmein2"
      }
    ]
  }
}
```

<!-- Please explain steps to reproduce -->

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

deboer-tim commented on packages/frontend/src/Build.svelte:
```diff
@@ -632,8 +638,32 @@ export function goToHomePage(): void {
+                <!-- AWS -->
                 <div>
-                  <span class="text-sm font-semibold mb-2 block">Upload image to AWS</span>
+                  <span class="text-md font-semibold mb-2 block">Upload image to AWS</span>
```
I don't think text-md is defined? May be just defaulting to text-base or whatever the parent is here.

cdrage commented:
![Screenshot 2024-07-10 at 8 24 23 AM](https://github.com/containers/podman-desktop-extension-bootc/assets/6422176/fd95650c-9f98-444a-a085-b63217574f62)

Updated the PR, I've also adjusted the classes a bit / styling so that the spacing is better for the AWS section as well.

Ready for another review 👍

deboer-tim commented:
> Ready for another review 👍

LGTM, previous approval still good.
